rails/benchmarks/bm_actioncontroller_api.rb

```diff
@@ -0,0 +1,16 @@
+require 'bundler/setup'
+require 'action_controller'
+require_relative 'support/benchmark_rails'
+
+class ApiController < ActionController::API
+  def show
+    render json: "JSON", status: :ok
```

Comment:
curiously enough, "JSON" isn't actually valid JSON (unless you're using some quirky mode processing like rails does by default). Maybe we should change this to `render plain: "TEXT", status: :ok`?